src/components/FormCheckbox.vue

```diff
@@ -119,15 +162,35 @@ export default {
 
             return key;
         },
-        preventOnReadonly(event) {
+        preventWhenReadonly(event) {
             if (this.input.readonly) {
                 event.preventDefault();
             }
         },
+        validate(scroll = false) {
+            this.errors = [];
+            if (
+                this.input.required
+                && (!this.value || this.value === '')
```

Comment:
ak bude value prázdny string tak vôjde už pri prvej podmienke.. lebo tá zahŕňa už aj druhú.. takže druhá je zbytočná

Reply:
a je to checkbox.. tam by si sa nemal pýtať na value ale na checked